Makefile

```diff
@@ -277,11 +279,23 @@ GOTESTSUM = $(shell pwd)/bin/gotestsum
 .PHONY: gotestsum
 gotestsum: ## Download gotestsum locally if necessary.
 	@GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on $(GO_CMD) install gotest.tools/gotestsum@v1.8.2
+
 KIND = $(shell pwd)/bin/kind
 .PHONY: kind
 kind:
 	@GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on $(GO_CMD) install sigs.k8s.io/kind@v0.16.0
 
+CODEGEN_ROOT = $(shell $(GO_CMD) env GOMODCACHE)/k8s.io/code-generator@$(CODEGEN_VERSION)
+.PHONY: code-generator
+code-generator:
+	@GO111MODULE=on $(GO_CMD) get \
```

Comment:
what happens if you use `install` instead?

Reply:
The errors occurred when we run https://github.com/kubernetes/code-generator/blob/v0.26.6/generate-groups.sh since we don't manage these packages by go module.

Reply:
I can not show the error log since the CI history losed. So I'm reproducing now.